submissions/EmmanuelIdiaghe/php/loopNum.php

```diff
@@ -0,0 +1,14 @@
+<?php
+#Using Loops
+   for ($i=100; $i <= 150; $i++) { 
```

Comment:
it says between `100` and `150`. It means `100` and `150` should not be includes